config/initializers/stripe.rb

```diff
@@ -0,0 +1,6 @@
+Rails.configuration.stripe = {
```

Comment:
Is there a specific reason you chose to do this? It doesn't seem to be used anywhere else so it seems like an unnecessary level of abstraction/configuration, especially since you have the values in the environment.